src/ducks/Alerts/TypeSteps/MetricsAndConditionsSelector/Conditions/LastPrice/LastPrice.js

```diff
@@ -0,0 +1,60 @@
+import React, { useMemo } from 'react'
+import gql from 'graphql-tag'
+import { useQuery } from '@apollo/react-hooks'
+
+import { formatNumber } from '../../../../../../utils/formatting'
+
+import styles from './LastPrice.module.scss'
+
+const LAST_PRICE_QUERY = gql`
+  query getMetric($slug: String!) {
+    getMetric(metric: "price_usd") {
+      timeseriesData(slug: $slug, from: "utc_now-1d", to: "utc_now") {
+        value
+        datetime
+      }
+    }
+  }
+`
+
+const useLastPrice = slugTitle => {
```

Comment:
`[D-0-7]` For function better to use the `function` declaration. We are using `const` for components, one line functions and variables